doc: Remove info about migrating active sessions, match the branding (#443)
TheThingsNetwork/docs: _content/the-things-stack/migrate-to-v3.md

## V3 login

> To be able to continue following steps below, you first need to log in to <a href="https://console.cloud.thethings.network/" target="_blank">The Things Stack development console</a>. You can do so by using your credentials after registering, or via The Things ID feature and use your known username and password from The Things Network.

Reviewer: ```suggestion
To be able to continue following steps below, you first need to log in to <a href="https://console.cloud.thethings.network/" target="_blank">The Things Stack Console</a>. You can do so by logging in with your The Things Network credentials via The Things ID.
```

### Note about migrating an active session

> Starting from The Things Stack version `v3.12`, for certain deployments it is possible to migrate active sessions as well. Migrating an active session means that OTAA devices will not have to perform a rejoin on the V3 network, and both OTAA and ABP devices will not have to be deleted from the V2 cluster in order to work properly when migrated to the V3 cluster. 

Reviewer: ```suggestion
Starting from The Things Stack version v3.12.0, for certain deployments it is possible to migrate active sessions as well. Migrating an active session means that OTAA devices will not have to perform a rejoin on the V3 network, and both OTAA and ABP devices will not have to be deleted from the V2 cluster in order to work properly when migrated to the V3 cluster. 
```

> **Important note:** Migrating active sessions is **not available for The Things Stack Community Edition**, i.e. you will not be able to migrate active sessions if you are using The Things Network. The reason for this is that The Things Stack Community Edition uses a different `DevAddr` block than the one used by The Things Network V2. 
>

> > However, migrating active sessions is available for other V3 deployments, including The Things Stack Open Source. Visit <a href="https://www.thethingsindustries.com/deployment/" target="_blank">Deployment Options</a> page to see how you can upgrade your plan, or <a href="mailto:support@thethingsindustries.com" target="_blank">contact our support</a>.

Reviewer: This shouldn't be buried in notes. Assume that the audience is The Things Network community. Session migration isn't available for them. You might want to add a note for V2 SaaS customers though, but that's it.

Author: @johanstokking I'm confused here, isn't it that the V2 SaaS (which was a paid offering, correct me if I'm wrong) customers can migrate active sessions but only to paid offerings of V3 (and not the Community Edition)? If yes, why is that relevant for TTN community docs?

Reviewer 2: I would suggest to create a headline about the session migration, and add this note about V2 SaaS customers. 

## Devices need to rejoin The Things Stack Community Edition
Migrating active sessions is not possible ...

### Note about migrating an active session

Starting from The Things Stack version `v3.12`, for certain deployments it is possible to migrate active sessions as well. Migrating an active session means that OTAA devices will not have to perform a rejoin on the V3 network, and both OTAA and ABP devices will not have to be deleted from the V2 cluster in order to work properly when migrated to the V3 cluster. 

> > **Important note:** Migrating active sessions is **not available for The Things Stack Open Source and The Things Stack Community Edition (The Things Network)**. Visit <a href="https://www.thethingsindustries.com/deployment/" target="_blank">Deployment Options</a> page to see how you can upgrade your plan, or <a href="mailto:support@thethingsindustries.com" target="_blank">contact our support</a>.

Reviewer 3: Importing device sessions should work for TTS Open Source. It is indeed not going to for TTNv2 -> The Things Stack Community Edition.

Maybe it would help to add the reasoning behind this (The Things Stack Community Edition is using a different DevAddr block from TTNv2, so even if the session is imported, device traffic will not go through.

Reviewer: https://thethingsnetwork.org/docs is not about TTSOS. The audience is TTN users.

### Adjust MAC settings for imported devices in V3

> If you have migrated an OTAA end device, i.e. your device performed a new join to the The Things Stack V3, the MAC settings will be automatically configured correctly by the Network Server. 

Reviewer 3: ```suggestion
If you have migrated an OTAA end device and your device performed a new join to the The Things Stack V3, the MAC settings will be automatically configured correctly by the Network Server. 
```

> Hovewer, if you have migrated an ABP device, you might want to configure the MAC settings manually. 

Reviewer 3: ```suggestion
However, if you have migrated an ABP device, you will have to configure the MAC settings manually. 
```
